src/components/upload/video-uploader.tsx

```diff
@@ -32,7 +32,7 @@ const VideoUploader = ({
         // filename with no extension
         const filename = fullFilename.replace(/\.[^/.]+$/, '')
         // remove stuff s3 hates
-        const scrubbed = `${filename}-${uuid.generate()}`
+        const scrubbed = `${filename}-${nanoid()}`
```

Comment:
Looking into shortid, I think the default was a 7 character identifier, so for consistency:

```suggestion
        const scrubbed = `${filename}-${nanoid(7)}`
```